### PR TITLE
Added Component badges/shields

### LINKED ISSUE #1313: Display a Badge for each Component
Support an endpoint for each component such that you can do things like:

 - Operational
 - Performance Issues
 - Partial Outage
 - Major Outage

I'm using http://shields.io/ as the example (which we could use as the implementation). 

I expect we could do something like:
 1. Go to https://demo.cachethq.io/api/components/1/badge
 2. Redirects with 302 Found (or 307 Temporary Redirect, your choice) to shields.io

We could also replace the text on the UI with these badges:
screen shot 2015-12-28 at 9 44 48 am

__Note:__ I would love to contribute this change.  Do you have recommendations for where this should go?  What should the messages be (status/name/etc.)?

### PR DESCRIPTION
**Pull request by jbrooksuk**
image

---

Closes #1313.

### CODE REVIEW

**Comment by jbrooksuk** [comment]
CC @CachetHQ/owners can we get this in?

**Comment by jbrooksuk** [comment]
We're serving with the right content-type now.

**Comment by joecohens** [comment]
:+1:

File: composer.lock [hunk 1]
```diff
@@ -2021,16 +2082,16 @@
         },
         {
             "name": "mccool/laravel-auto-presenter",
-            "version": "4.2.1",
+            "version": "v4.2.1",
```
Comment (GrahamCampbell) [inline composer.lock]: @jbrooksuk Please clear your composer cache. Composer has f***ed up here again.

Comment (joecohens) [inline composer.lock]: This is odd

Comment (GrahamCampbell) [inline composer.lock]: Not really. It was my fault. I messed the name and retagged, but composer doesn't bother with that and uses it's stale cache.

Comment (jbrooksuk) [inline composer.lock]: Ah ok :)

**Pull request merged**